### ISSUE TITLE
schedule_dependency_monkey() got an unexpected keyword argument 'requirements'

### ISSUE THREAD

**Issue by harshad16** [thoth-station/management-api#726]
**Describe the bug**
```
File \"/opt/app-root/lib64/python3.8/site-packages/flask/app.py\", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File \"/opt/app-root/lib64/python3.8/site-packages/flask/app.py\", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File \"/opt/app-root/lib64/python3.8/site-packages/flask/app.py\", line 1821, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File \"/opt/app-root/lib64/python3.8/site-packages/flask/_compat.py\", line 39, in reraise
    raise value
  File \"/opt/app-root/lib64/python3.8/site-packages/flask/app.py\", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File \"/opt/app-root/lib64/python3.8/site-packages/flask/app.py\", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File \"/opt/app-root/lib64/python3.8/site-packages/connexion/decorators/decorator.py\", line 48, in wrapper
    response = function(request)
  File \"/opt/app-root/lib64/python3.8/site-packages/connexion/decorators/uri_parsing.py\", line 144, in wrapper
    response = function(request)
  File \"/opt/app-root/lib64/python3.8/site-packages/connexion/decorators/validation.py\", line 184, in wrapper
    response = function(request)
  File \"/opt/app-root/lib64/python3.8/site-packages/connexion/decorators/validation.py\", line 384, in wrapper
    return function(request)
  File \"/opt/app-root/lib64/python3.8/site-packages/connexion/decorators/parameter.py\", line 121, in wrapper
    return function(**kwargs)
  File \"/opt/app-root/src/thoth/management_api/api_v1.py\", line 244, in post_dependency_monkey_python
    return _do_schedule(
  File \"/opt/app-root/src/thoth/management_api/api_v1.py\", line 585, in _do_schedule
    \"analysis_id\": runner(**parameters, **runner_kwargs),
TypeError: schedule_dependency_monkey() got an unexpected keyword argument 'requirements'"}
```

Related-to: #725

**Comment by harshad16** [comment]
Observed in v0.14.0 management-api
fixed in the v0.15.0

**Comment by fridex** [comment]
Thanks, the thoth-common update for optimizing data flow. PR https://github.com/thoth-station/thoth-application/pull/982 should fix the issue.
